src/Microsoft.Diagnostic.TestHelpers/Microsoft.Diagnostic.TestHelpers.csproj

```diff
@@ -1,21 +1,21 @@
-﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>;1591;1701</NoWarn>
+    <IsShipping>false</IsShipping>
     <IsPackable>true</IsPackable>
     <Description>Diagnostic test support</Description>
     <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
     <PackageTags>tests</PackageTags>
     <DebugType>embedded</DebugType>
-    <DebugSymbols>true</DebugSymbols>
```

Comment:
What does DebugSymbols do?  I probably cut and pasted it without knowing or assuming it was needed to generate symbols/pdbs.

Reply:
Automatically generates PDBs for all builds. However, this is implicit in Arcade and I hoisted out to be true for all projects